source/context_test.go

```diff
@@ -21,7 +21,7 @@ func TestSource_FromContext(t *testing.T) {
 	context, engine := gin.CreateTestContext(nil)
 
 	// setup mock server
-	engine.GET("/health", func(c *gin.Context) { c.String(http.StatusOK, "OK") })
+	engine.GET("/health", func(c *gin.Context) { c.String(http.StatusOK, "OK-") })
```

Comment:
just curious what this change is for?

Reply:
Added to test returns. I forgot to remove them.